### DISCUSSION TITLE
Installation fix

### DISCUSSION THREAD

**Discussion by IngridJSJ** [Spine-project/Spine-Toolbox#1975] [Show and tell]
Hi! 
After installing spinetoolbox as its described in the readme I got this error in the importer:
Could not connect to source: 'ReadOnlyWorksheet' object has no attribute 'defined_names'
The fix was to uninstall the openpyxl==3.1.0 in the spinetoolbox environment and  change it to openpyxl==3.1.0

**Comment by ptsavol** [comment]
I upgraded my openpyxl to 3.1.1 but I don't see any problems. I did execute some project that uses Excel files and the unit tests pass but it is possible that some problem may be lurking somewhere. Can anybody else confirm a problem with openpyxl 3.1.1?
